static/js/vue-cdr-access/src/components/filterTags.vue

```diff
@@ -99,27 +93,36 @@ export default {
             if (fieldValue === undefined || TYPES[type] === undefined) {
                 return '';
             }
-            let display_text = decodeURIComponent(fieldValue);
-
-            // Format time based tags
-            if (type === 'added' || type === 'created' || type === 'createdYear') {
-                display_text = this._formatTime(fieldValue);
-            }
-
-            let tag_info = {
-                type: type,
-                type_text: TYPES[type],
-                original_value: fieldValue,
-                value_text: display_text
-            };
-
-            // Return non multi-value tags
-            if (!MULTI_VALUED_FIELDS.includes(type)) {
+            if (Array.isArray(fieldValue)) {
+                // Multivalued fields
+                return fieldValue.map((value) => {
+                    let tag_info = {
```

Comment:
Could probably move this out of the if statement, since both fields repeat in the else statement. Then in the else clause  just set the original_value `tag_info.original_value = fieldValue`